### PR TITLE
[CCM-13907]: Anomaly docs

### PR DESCRIPTION
**Pull request by joeyouss**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [X] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screenshot.

### CODE REVIEW

File: docs/cloud-cost-management/3-use-ccm-cost-reporting/4-detect-cloud-cost-anomalies-with-ccm.md [hunk 1]
```diff
@@ -8,14 +8,22 @@ helpdocs_is_private: false
...
 
 Currently, CCM detects anomalies for the following:
 
-* **Clusters**: Kubernetes Clusters, Namespaces, and Workloads
-* **AWS**: Service and Account
-* **GCP**: Products, Projects, and SKUs
-* **Azure**: Subscription ID, Service Name, and Resources
+ **Clusters**: Kubernetes clusters, namespaces, and workloads
+ * **AWS**: Services, accounts, and usage types
+ * **GCP**: Products, projects, and SKUs
```
Comment (shashwat-harness) [inline docs/cloud-cost-management/3-use-ccm-cost-reporting/4-detect-cloud-cost-anomalies-with-ccm.md:19]: Write in order of parent to child entity i.e "Projects, products and SKUs"

File: docs/cloud-cost-management/3-use-ccm-cost-reporting/4-detect-cloud-cost-anomalies-with-ccm.md [hunk 2]
```diff
@@ -8,14 +8,22 @@ helpdocs_is_private: false
...
+ Harness CCM detects cost anomalies for your Kubernetes clusters and cloud accounts. Cloud cost anomaly detection can be used as a tool to keep cloud costs under control. It also provides alerting capabilities through email and Slack to notify stakeholders when an anomaly is detected.
 
 Currently, CCM detects anomalies for the following:
 
-* **Clusters**: Kubernetes Clusters, Namespaces, and Workloads
-* **AWS**: Service and Account
-* **GCP**: Products, Projects, and SKUs
-* **Azure**: Subscription ID, Service Name, and Resources
+ **Clusters**: Kubernetes clusters, namespaces, and workloads
+ * **AWS**: Services, accounts, and usage types
```
Comment (shashwat-harness) [inline docs/cloud-cost-management/3-use-ccm-cost-reporting/4-detect-cloud-cost-anomalies-with-ccm.md:18]: Write in order of parent to child entity i.e "Accounts, services and usage types"

File: docs/cloud-cost-management/3-use-ccm-cost-reporting/4-detect-cloud-cost-anomalies-with-ccm.md [hunk 3]
```diff
@@ -8,14 +8,22 @@ helpdocs_is_private: false
...
+ * **GCP**: Products, projects, and SKUs
+ * **Azure**: Subscription IDs, service names, and resources
```
Comment (shashwat-harness) [inline docs/cloud-cost-management/3-use-ccm-cost-reporting/4-detect-cloud-cost-anomalies-with-ccm.md:20]: "Subscription IDs, resource groups and meter categories"

File: docs/cloud-cost-management/3-use-ccm-cost-reporting/4-detect-cloud-cost-anomalies-with-ccm.md [hunk 4]
```diff
@@ -57,11 +109,28 @@ For example, the difference between the actual and predicted is 120 / 21 = 5.71
...
+
+This is a time series forecasting model designed to handle the challenges often encountered in time series data, such as seasonality, holidays, and other recurring patterns. The model combines both additive and multiplicative elements to effectively capture the diverse patterns present in the data.
+
+## Data requirement for anomaly prediction
+The time duration considered for each entity ranges from a minimum of 14 days to a maximum of 36 days, with these values currently being set as the boundaries. For the algorithms that analyze these time series data, a minimum of 14 days of data is needed to effectively grasp usage patterns and make predictions based on those patterns. However, a problem arises when new services or dimensions are added to the system. These newly introduced elements don't yet have the required 14 days worth of data for analysis. To address this challenge, CCM extends the data timeline by adding zeros to fill the gaps, ensuring that a continuous 2-week period of data is available. With this extended data, the algorithms can then accurately predict anomalies even for the new entities until a full 14 days of real data is gathered.
```
Comment (shashwat-harness) [inline docs/cloud-cost-management/3-use-ccm-cost-reporting/4-detect-cloud-cost-anomalies-with-ccm.md:123]: minimum of 14 days to a maximum of 41 days

**Review by shashwat-harness — DISMISSED** [review]

File: docs/cloud-cost-management/3-use-ccm-cost-reporting/4-detect-cloud-cost-anomalies-with-ccm.md (hunk 4 above)
Comment (joeyouss) [reply docs/cloud-cost-management/3-use-ccm-cost-reporting/4-detect-cloud-cost-anomalies-with-ccm.md]: done

File: docs/cloud-cost-management/3-use-ccm-cost-reporting/4-detect-cloud-cost-anomalies-with-ccm.md (hunk 3 above)
Comment (joeyouss) [reply docs/cloud-cost-management/3-use-ccm-cost-reporting/4-detect-cloud-cost-anomalies-with-ccm.md]: done

**Pull request merged**